.travis/before_install.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-yarn global add lerna@3.13.4
+# yarn global add lerna@3.13.4
```

Comment:
we needed this in the past? Is that not the case anymore?

Reply:
I swapped the travis commands to just use the root package.json scripts 👍

Reply:
So let's remove that line completely then?

Reply:
I can remove the commented out command

Reply:
Sorry, submitted after you commented ... I'll do that 👍